guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -4,9 +4,6 @@
 The Discovery image is a minimal operating system that is PXE-booted on hosts to acquire initial hardware information and to check in with {Project}.
 Discovered hosts keep running the Discovery image until they are rebooted into Anaconda, which then initiates the provisioning process.
 
-ifdef::satellite[]
-The operating system image is based on {RHEL} 7.
-endif::[]
```

Comment:
This should remain. I hope a new version is built on RHEL 8, but that should be confirmed.